facebookads/objects.py

```diff
@@ -487,6 +488,9 @@ def remote_create(
         params = {} if not params else params.copy()
         params.update(self.export_data())
 
+        # some private endpoints require a custom relative_path
+        relative_path = relative_path or (self.get_parent_id_assured(), self.get_endpoint())
```

Comment:
Do we still need this change? I thought this was only in the effort to support `DELETE`